learnStrings/learnString.test.js

```diff
@@ -0,0 +1,16 @@
+import { findMostRepChar, checkPalindrome, removeLastPath } from './learnString'
+
+test("find most repeated char in the array", () => {
+    expect(findMostRepChar("abb")).toEqual(["b"])
+    expect(findMostRepChar("abbcc")).toEqual(["b", "c"])
```

Comment:
Use proper match to check for repeated char.

Reply:
done